src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/FAA/FaaApplicationStatusSummary.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services.FAA
+{
+    public class FaaApplicationStatusSummary
+    {
+        public FaaApplicationStatus ApplicationStatus { get; set; }
+        public string UnsuccessfulReason { get; set; }
+        public bool IsRecruitVacancy { get; set; }
+        public int? VacancyReference { get; set; }

```

Comment:
In V2, we use long for vacancyReference, looking at FAA/VSS code, it seems this is a property you have added to ApplicationStatusSummary, could it be a string instead as casting a long to an int could cause issues down the line even though we are not likely to break int32 yet for our vacancyReference nos.

Reply:
VacancyId across FAA is an `int`. Our VacancyReference is mapped to this `int` field.  So in order to integrate with FAA we need to be talking `int`.  So I want this to be turned into an `int` (and potentially fail) as soon as possible on our side before we send messages to FAA.

If we can no longer cast to an `int` we'll have a bigger architecture problem across FAA.